src/lib/index.ts

```diff
@@ -1,7 +1,32 @@
 import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
-import { specmaticJarPathLocal } from '../config';
+import { specmaticJarPathLocal, specmatic } from '../config';
+import fs from 'fs';
+import { check } from 'yargs';
+
+type Environment = Record<string, string>
+export const setSpecmaticEnvironement = (environmentName: string, environment: Environment) => {
+  console.log("Reading the file from -> ", path.resolve(specmatic))
+  let file = null
+  let flag = false
+  try {
+    file = require(path.resolve(specmatic))
+    try {
+      for (var x in environment) {
```

Comment:
Looping over each element in the object means, that many io operations. Instead we can prepare the full contents and write them once

Reply:
Made the required changes